@navikt/core/react/src/layout/box/Box.tsx

```diff
@@ -99,22 +100,28 @@ export const Box: OverridableComponent<BoxProps, HTMLDivElement> = forwardRef(
     },
     ref,
   ) => {
+    const themeContext = UNSAFE_useAkselTheme(false);
+    const prefix = themeContext ? "ax" : "a";
+
     const style: React.CSSProperties = {
       ..._style,
-      "--__ac-box-background": background
+      [`--__${prefix}-box-background`]: background
```

Comment:
You have used a/ax prefix several places that previously had "ac" prefix. This is just one example.

Reply:
[Think i resolved most of it in this PR](https://github.com/navikt/aksel/pull/3423) 🤞 That merges into this PR